pkg/volume/util/operationexecutor/operation_executor.go

```diff
@@ -753,12 +754,12 @@ func (oe *operationExecutor) VerifyVolumesAreAttached(
 				volumeSpecMapByPlugin[pluginName] = volumeSpecMap
 				continue
 			}
-
 			// If node doesn't support Bulk volume polling it is best to poll individually
-			nodeError := oe.VerifyVolumesAreAttachedPerNode(nodeAttachedVolumes, node, actualStateOfWorld)
-			if nodeError != nil {
-				klog.Errorf("VerifyVolumesAreAttached failed for volumes %v, node %q with error %v", nodeAttachedVolumes, node, nodeError)
-			}
+			needVerifyVolumes = append(needVerifyVolumes, volumeAttached)
+		}
+		nodeError := oe.VerifyVolumesAreAttachedPerNode(needVerifyVolumes, node, actualStateOfWorld)
```

Comment:
Is it possible to have a unit test for this behavior that could count the number of calls and/or the volumes that were checked?

Reply:
Sorry for delay!
I am afraid can not  count the number  with UT.